### ISSUE TITLE
The center loss is very large

### ISSUE THREAD

**Issue by SuyueLiu** [KaiyangZhou/pytorch-center-loss#26]
At the beginning of training, the center loss is very large, for example, 520, although the use of pre-trained model.
PS. I use the arcface as pre-train before adding center loss
